### PR TITLE
Improve navbar and sidebar recipes

### PR DESCRIPTION
**Pull request by picklelo**

### CODE REVIEW

File: docs/recipes/navbar.md [hunk 1]
```diff
@@ -16,22 +16,136 @@ In this example we want the navbar to stick to the top of the page, so we will u
  prop to make the navbar fixed to the top of the page.
 We will also use the `top= 0` and `z_index="1"` props to make sure the navbar is always on top of the screen and above the other components on the page.
```
Comment (tgberkeley) [inline docs/recipes/navbar.md:17]: say `z_index="1"` in the text but then have `z_index="5"` in the code. Also maybe quickly explain what the z_index and how it makes sure that it is always at the top

File: docs/recipes/navbar.md [hunk 2]
```diff
@@ -16,22 +16,136 @@ In this example we want the navbar to stick to the top of the page, so we will u
...
+        rx.heading("Welcome to My App"),
+        rx.text("This is the main content of the page."),
+    )
+
+def index():
+    return rx.fragment(
+        navbar(),
+        rx.container(
+            content(),
+            padding_top="6em",
```
Comment (tgberkeley) [inline docs/recipes/navbar.md:96]: How do we know this is a good amount of padding to add, is this trial and error or is there a better way to know?

Comment (picklelo) [reply docs/recipes/navbar.md:100]: Added a note on this

File: docs/recipes/sidebar.md [hunk 3]
```diff
@@ -8,25 +8,153 @@ Similar to a navigation bar, a sidebar is a common UI element found on the side
 In this recipe, we will create a sidebar component than can help with navigation in a web app.
 
 In this example we want the sidebar to stick to the left side of the page, so we will use the `position="fixed"` prop to make the sidebar fixed to the left side of the page.
-We will also use the `left=0` and `z_index=1` props to make sure the sidebar is always on top of the screen and above the other components on the page.
+We will also use the `left=0`, `top=0`, and `z_index=5` props to make sure the sidebar is always on top of the screen and above the other components on the page.
```
Comment (tgberkeley) [inline docs/recipes/sidebar.md:11]: comment on what these left and top props are

**Review by tgberkeley — APPROVED** [review]

**Pull request merged**